_docs/technical/2017-02-12-time.md

```diff
@@ -0,0 +1,89 @@
+
+Nodes use NTP-servers for synchronization, read [What is NTP?](http://www.ntp.org/ntpfaq/NTP-s-def.htm) for more details. So, this is a core idea of NTP-client and NTP-server communication:
+
+~~~
+   NTP-client                     NTP-server
+
++---------------+              +---------------+
+|               |              |               |
+|      sent REQ ---> A    X ---> received REQ  |
+|               |              |               |
+| received RESP <--- B    Y <--- sent RESP     |
+|               |              |               |
++---------------+              +---------------+
+~~~
+
+where:
+
+1. `REQ` - time request (NTP message).
+2. `RESP` - time response (NTP Data Packet).
+3. `A` - timestamp when the client actually sent `REQ` to the server.
+4. `X` - timestamp when the server actually received `REQ` from the client.
+5. `Y` - timestamp when the server actually sent `RESP` to the client.
+6. `B` - timestamp when the client actually received `RESP` from the server.
+
+Timestamp consists of seconds (32 bit) and second fractions (32 bit), since date **01.01.1900**.
+For example, date "1 January 2000: corresponds to seconds value `3155673600`. Second fraction value 
+can be different, up to minimal possible value 232 picoseconds.
+
+Please note that timestamps `A` and `B` are "wrong" time because it's obtained from the client's machine, and we cannot trust these timestamps by default. Timestamps `X` and `Y` are "true" time because it's obtained from the server.
+
+We can imagine it on a time scale with these example values (in seconds):
+
+~~~
+----*-------*-------------*-------*---->
+    A       X             Y       B
+    100     150           160     120
+~~~
+
+It's assumed that network delays are equal (i.e time for transmitting `REQ` from the client to the server and time for transmitting `RESP` from the server to the client are the same). So from the client's point of view total transmit time is `B` - `A`, e.g. 120 - 100 = 20 seconds. `RESP` contains `X` and `Y`, so client can calculate server's delay as `Y` - `X`, e.g. 160 - 150 = 10 seconds, let's call it `D`. 20 - `D` = 10 seconds, it's total network delay, 10 / 2 = 5 seconds, it's amount of time it took the response to get from the server to the client, let's call it `R`.
+
+Then NTP-client adds `R` to the `Y` to estimate that it received the response at "true" time 165 seconds, let's call it `T`. So `T` - `B`, e.g. 165 - 120 = +45 seconds, let's call it `C`, it's a correction NTP-client has to make with it's local time.
+
+## Cardano SL Time Concepts
+
+There're fundamental time concepts we use in our implementation:
+
+1. **global time** - time obtained from NTP server. We can think of it as of "true" time.
+2. **local time** - [current time](https://github.com/input-output-hk/cardano-sl/blob/f754d7a48294d7c434d401af12075dbe3fa4a7a9/infra/Pos/Slotting/Ntp.hs#L294) obtained from the node. We can think of it as of "wrong" time, because it can actually be wrong.
```

Comment:
@denisshevchenko these two definitions are shaky, refactor them. Perhaps @staenrey can help you express what you want with better words.